driver/src/main/java/org/dcache/SapphireDriver.java

```diff
@@ -65,6 +80,61 @@ public SapphireDriver(String type, String name)
         executorService = new ScheduledThreadPoolExecutor(1, threadFactory);
     }
 
+    @Override
+    public void start() {
+        LOGGER.debug("Triggered start()");
+        try{
+            mongoClient = MongoClients.create(mongoUri);
+        } catch (MongoSocketOpenException e) {
+            LOGGER.error("Could not open connection to MongoDB");
+            throw e;
+        }
+        MongoDatabase mongoDatabase = mongoClient.getDatabase(this.database);
+        files = mongoDatabase.getCollection("files");
+        stageFiles = mongoDatabase.getCollection("stage");
+
+        try {
+            server = new FileServer(hostname, port, whitelist, certfile, keyfile);
+            server.startServer();
+        } catch (Exception e) {
+            LOGGER.error("Could not start Jetty server", e);
+            throw new RuntimeException(e);
```

Comment:
this will kill dcache as well. Is I desired?

Reply:
No, it should just stop the hsm plugin from being created